core/modules/system/config/system.core.json

```
@@ -80,6 +80,9 @@
     "user_mail_status_canceled_notify": 0,
     "user_register": "admin_only",
     "user_signatures": 0,
+    "user_password_reject_weak": 0,
+    "user_password_min_length_constraint": 0,
```

I still feel that this has the risk of being confused with the min length number:

```suggestion
    "user_password_min_length_constraint": 0,
```

I preferred it with the word `_enabled`, which clearly implies that values `1`/`0` for on/off are to be expected, and these are not string length.

---

Cool